web_editor/js/main.js

```diff
@@ -0,0 +1,147 @@
+// TODO The hash changes, this is very brittle. See
+// https://github.com/thedodd/trunk/issues/230 or stop using trunk.
+import { dummyData } from "./dummy_data.js";
+import init, {
+  js_way_to_lanes,
+  js_lanes_to_tags,
+} from "./osm2lanes-npm/osm2lanes_npm.js";
+await init();
+
+// Setup the toolbox controls
+new Sortable(document.getElementById("toolbox"), {
+  group: {
+    name: "toolbox",
+    pull: "clone",
+  },
+  sort: false,
+  animation: 150,
+  ghostClass: "card-being-dragged",
+});
+
+// TODO Disable the ghostClass of lanes when going here
+new Sortable(document.getElementById("delete"), {
+  group: "lanes",
+  onAdd: function (evt) {
+    var el = evt.item;
+    el.parentNode.removeChild(el);
+  },
+});
+
+// This state is set up by start_editing
```

Comment:
The state management is partly here. These vars represent the original version of the currently fetched way. Then the "current" version of the lanes exists in the DOM, within the ordered `cards` div. Each card has a `laneJSON` attribute smushed into it. I think this lets us avoid any hassle of syncing the DOM with state kept elsewhere. Is this a reasonable pattern?